app/views/invitations/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+
+<%= simple_form_for [@cycle, @invitation], url: create_invitation_path do |f| %>
```

Comment:
If you have used nested resources in your routers file, you could use something like `cycle_invitation_path`. You can run `rake routes` to double check the exact name of the path helper